src/libs/ReactNativeLauncher.js

```diff
@@ -410,6 +411,7 @@ class ReactNativeLauncher extends Launcher {
       throw new Error(`worker bridge restart init error: ${err.message}`)
     }
     this.emit('WORKER_RELOADED')
+    this.emit('worker:webview:ready')
   }
```

Comment:
> but it fails when running goto twice in a row for example 

Can you add a test for that please?

Reply:
Since this is at least the second time in a row we have an issue on this case, we should add a lost of tests!

Reply:
You can just test the ReactNativeLauncher, without the LauncherView. We can test that the events are well emitted. 

But OK to do that later.